angular/components/README.md

```diff
@@ -0,0 +1,153 @@
+# Components
+
+This page will give you some practices and tips/tricks about Angular components.
+
+## Clean Up Component to Prevent Memory Leak
+
+For example, when you create `interval` observable
+
+```ts
+import { interval, Subscription } from 'rxjs';
+
+@Component({...})
+export class MyComponent implements OnInit, OnDestroy {
+    ngOnInit () {
+        // create interval observable
+        const source = interval(60000);
+        source.subscribe(...);
+    }
+}
+```
+
+You should remember your observable and call `unsubscribe` function when component destroyed.
```

Comment:
Can we recommend them to use `subsink` krubb?

https://c0d1um.github.io/technical-standard/angular/third-party-libraries/#utilities

Reply:
yes it is better.